vector/src/main/java/im/vector/app/features/home/ShortcutCreator.kt

```diff
@@ -65,6 +66,10 @@ class ShortcutCreator @Inject constructor(
                 .setShortLabel(roomSummary.displayName)
                 .setIcon(bitmap?.toProfileImageIcon())
                 .setIntent(intent)
+
+                // Make it show up in the direct share menu
+                .setCategories(setOf(directShareCategory))
```

Comment:
This is maybe not the ideal place to add this code. `create()` is called the user create a shortcut, or for the first 4 favorite rooms.

I think other apps create direct share shortcut once something has been shared, for a quicker future sharing to the same target.

Reply:
Ok, maybe I have misunderstood this. I think it is OK actuallly

Reply:
Such a ranking is still done on the android side (of course only with the shortcuts published to them).

Reply:
Yes, my remark was just about what will be the rooms exposed here